Makefile

```diff
@@ -96,8 +99,13 @@ prepare-minor:
 prepare-major:
 	./hack/prepare-release.sh major
 
+$(GITHUB_RELEASE): go.mod
+	go install ./vendor/github.com/aktau/github-release
+
 release: IMAGE_TAG = $(shell hack/version.sh)
 release: docker-build docker-push
+release: $(GITHUB_RELEASE)
+	TAG=$(IMAGE_TAG) GITHUB_RELEASE=$(GITHUB_RELEASE) DESCRIPTION=./version/description ./hack/release.sh
```

Comment:
The `description` part is not working ... Does anything scream bloody murder there, @phoracek  ?

Reply:
My bad!!! It says `empty token`, which I'm assuming is the missing github token.